download/es/download-easy.wml

```diff
@@ -52,7 +75,7 @@
 	      (<a class="win-tbb-sig" href="../dist/torbrowser/<version-torbrowserbundle>/torbrowser-install-<version-torbrowserbundle>_<lang>.exe.asc">firma</a>) <a class="siginfo" href="<page docs/verifying-signatures>">¿Que es Esto?</a>
 	    </div>
 	  </form>
-	  <p class="alt-dl">¿No usas Windows?<br />Descargalo para <a href="#mac">Mac</a> o <a href="#linux">Linux</a></p>
+	  <p class="alt-dl">¿No usas Windows?<br />Descargalo para <a href="#mac">Mac</a> or <a href="#linux">Linux</a></p>
```

Comment:
Please, change this to "o" ("or" in spanish)

Reply:
done